### PR TITLE
Fix community plugins feedback

### PR DESCRIPTION
**Pull request by dbuzhorkaa**

### CODE REVIEW

File: layouts/plugins/single.html [hunk 1]
```diff
@@ -17,29 +17,67 @@
...
+        </div>
+      </div>
     </div>
   </div>
   <div class="container">
     <div class="row">
       <div class="col-12">
-        <h2 class="plugins__title"></h2>
+        <div class="plugins__quantity-wrapper" style="display: none">
+          <span class="plugins__quantity"></span> Plugin displayed
```
Comment (bdmogal) [inline layouts/plugins/single.html]: somehow the space doesn't show up in the rendering:
image

Also, lower case ``p`` and make plural: ``plugins``

File: data/en/pipelines.json [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+          "header": "Improved data trustworthiness",
+          "paragraph": "Pipelines offers data quality libraries, metadata and lineage capture, audit logging.",
+          "id": "improved-visibility"
+        }
+      ]
+    }
+  ],
+  "features_pipeline": [
+    {
+      "title": "Pipeline features",
```
Comment (bdmogal) [inline data/en/pipelines.json]: somehow, preparation features show up for pipelines:
image

File: layouts/plugins/single.html (hunk 1 above)
Comment (geksilla) [reply layouts/plugins/single.html]: done

File: data/en/pipelines.json (hunk 2 above)
Comment (geksilla) [reply data/en/pipelines.json]: done. Please recheck.

Comment (dbuzhorkaa) [reply data/en/pipelines.json]: Done. Please recheck

File: layouts/plugins/single.html (hunk 1 above)
Comment (dbuzhorkaa) [reply layouts/plugins/single.html]: done.

**Pull request merged**